lib/ios-deploy.js

```diff
@@ -192,7 +193,7 @@ class IOSDeploy {
     }
   }
 
-  async terminateApp(bundleId) {
+  async terminateApp(bundleId, platformVersion) {
```

Comment:
I would say it makes sense to keep platformVersion as IOSDeploy property as it might be useful in other methods as well

Reply:
Yea, that sounds good. let me do it as a follow-up pr (to pr split)